.github/workflows/generate-deb-packages.yaml

```diff
@@ -0,0 +1,20 @@
+name: generate-deb-packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  generate-deb-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jspricke/ros-deb-builder-action@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ROS_DISTRO: humble
+          DEB_DISTRO: jammy
+          SBUILD_CONF: |
+            use Dpkg::Build::Info;
+            $environment_filter = [Dpkg::Build::Info::get_build_env_allowed(), 'ROS_DISTRO'];
```

Comment:
```suggestion
```

This is not needed.

Reply:
When I tried, I got an error because `autoware_cmake` references ROS_DISTRO in CMakeLists.txt.

Reply:
That was probably before this was added: https://github.com/autowarefoundation/autoware_common/blob/main/autoware_cmake/package.xml#L15

I tested it and it works fine now.

Reply:
That's nice.